.github/workflows/contributor-pr.yml

```diff
@@ -33,9 +33,9 @@ jobs:
         with:
```

Comment:
While you are at it, should we add the concurrency settings as well? https://github.com/gradle/gradle/issues/19173

Reply:
PTAL!